### PR TITLE
carrying show argument forward to class functions and co

### PR DESCRIPTION
**Pull request by cmoutard**
Following  [#1984](https://github.com/mne-tools/mne-python/pull/1984), show argument is carried forward to class functions and co.

### CODE REVIEW

File: mne/epochs.py [hunk 1]
```diff
@@ -632,6 +635,8 @@ def plot_psd_topomap(self, bands=None, vmin=None, vmax=None, proj=False,
             function that returns patches (required for multi-axis plots).
         verbose : bool, str, int, or None
             If not None, override default verbose level (see mne.verbose).
+        show : bool
+            Show figure if True.
```
Comment (Eric89GXL) [inline mne/epochs.py]: by convention `verbose` is always last, so put `show` before it

File: mne/epochs.py [hunk 2]
```diff
@@ -642,7 +647,7 @@ def plot_psd_topomap(self, bands=None, vmin=None, vmax=None, proj=False,
             self, bands=bands, vmin=vmin, vmax=vmax, proj=proj, n_fft=n_fft,
             ch_type=ch_type, n_overlap=n_overlap, layout=layout, cmap=cmap,
             agg_fun=agg_fun, dB=dB, n_jobs=n_jobs, normalize=normalize,
-            cbar_fmt=cbar_fmt, outlines=outlines, verbose=None)
+            cbar_fmt=cbar_fmt, outlines=outlines, verbose=None, show=show)
```
Comment (Eric89GXL) [inline mne/epochs.py]: ... and if `show` doesn't come before `verbose` in `plot_psd_topomap`, please fix that, too.

**Comment by Eric89GXL** [comment]
Thanks! Were these the only remaining instances you could find that needed `show`?

**Comment by cmoutard** [comment]
using grep -R exhaustively on all the funcitons missing show argument, yes !

**Comment by Eric89GXL** [comment]
Great, thanks @cmoutard!

**Pull request merged**